### PR TITLE
JBPM-5757 - Replace status List<Integer> with List<Status> in CaseRunt…

### PR DESCRIPTION
**Pull request by tsurdilo**
…imeDataService operations

### CODE REVIEW

**Comment by tsurdilo** [comment]
jenkins retest this

**Comment by tsurdilo** [comment]
jenkins retest this

**Comment by livthomas** [comment]
The commit message should reference [JBPM-5757](https://issues.jboss.org/browse/JBPM-5757) instead of [JBPM-575](https://issues.jboss.org/browse/JBPM-575).

File: jbpm-case-mgmt/jbpm-case-mgmt-api/src/main/java/org/jbpm/casemgmt/api/model/CaseStatus.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+public enum CaseStatus {
+    PENDING,
+    OPEN,
+    CLOSED,
+    CANCELLED,
+    SUSPENDED
```
Comment (mswiderski) [inline jbpm-case-mgmt/jbpm-case-mgmt-api/src/main/java/org/jbpm/casemgmt/api/model/CaseStatus.java]: I would remove PENDING and SUSPENDED as they do not make much sense for cases at the moment - I mean we don't have support for it

**Comment by tsurdilo** [comment]
@livthomas why?

**Comment by tsurdilo** [comment]
@mswiderski amended..associated pr in kie-server - https://github.com/kiegroup/droolsjbpm-integration/pull/857

**Pull request merged**